snapper.py

```diff
@@ -679,9 +709,11 @@ def main():
         }
 
         email_report = create_email_report(report_data)
-
         send_email('SnapRAID Job Completed Successfully', email_report)
 
+        apprise_report = create_apprise_report(report_data)
+        send_apprise('SnapRAID Job Completed Successfully', apprise_report)
```

Comment:
Shouldn't these be guarded to ensure the feature is enabled?

Reply:
To be honest, I followed your send_mail implementation.  This relies on the enabled flag within the routine to guard.  It won't execute if not enabled:

https://github.com/brott8/snapper/blob/0c9d7d0e8ec400951bde7e9e813fe7892c71d8f2/snapper.py#L190

The main downside of this approach is that the template execution runs every time regardless.  I can suggest bringing the enabled flag into the main routine for both email and apprise, following your discord implementation.